include/class-polylang.php

```diff
@@ -189,6 +189,7 @@ public function init() {
 		 * @param string $class either PLL_Model or PLL_Admin_Model
 		 */
 		$class = apply_filters( 'pll_model', PLL_SETTINGS || self::is_wizard() ? 'PLL_Admin_Model' : 'PLL_Model' );
+		/** @var PLL_Model $model */
```

Comment:
Shouldn't we check the type of the object returned instead? Since it comes from a filter ...

Reply:
As we discussed, it's not useful.